setup.py

```diff
@@ -122,8 +128,7 @@ class MyBuildExt(build_ext):
     amalgamation = False
 
     def _pkgconfig(self, flag, package):
-        status, output = commands.getstatusoutput("pkg-config %s %s" % (flag, package))
```

Comment:
This was giving me "{ is not a recognized command" or something like that under msys. Looked up getstatusoutput (https://docs.python.org/2/library/commands.html) and they say this:
> cmd is actually run as { cmd ; } 2>&1, so that the returned output will contain output or error messages
So that would be where the "{" "not found" comes from.